src/Event/Productions/SimilaritiesClient.php

```diff
@@ -0,0 +1,58 @@
+<?php declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Event\Productions;
+
+use GuzzleHttp\Client;
+
+class SimilaritiesClient
+{
+    /**
+     * @var Client
+     */
+    private $client;
+
+    /**
+     * @var string
+     */
+    private $uri;
+
+    /**
+     * @var string
+     */
+    private $key;
+
+    public function __construct(Client $client, string $uri, string $key)
+    {
+        $this->client = $client;
+        $this->uri = $uri;
+        $this->key = $key;
+    }
+
+    public function markAsLinked(array $tuples): void
```

Comment:
Since this is the api client for GCP, we should probably use their terminology in our methods.
The way I think about this, we perform a link action which results in multiple event pairs being linked in a production. As a side effect, we call `->excludeTemporarily()` on the Similarities API.

Similarly, when we perform a `skip` action, an event pair is skipped and we should call an `->excludePermanently()` on the Similarities API.